docs/api/modules.rst

```diff
@@ -0,0 +1,7 @@
+track
```

Comment:
yeah this isn't quite right.. 

for one this folder I dont think needs to be committed. 
for two why is this only listing a single module instead of multiple. hmmm. hypothesis is that I think it's showing the most recently generated one (being overwritten in for-loop)